__tests__/nc_news_test.test.js

```diff
@@ -253,4 +253,82 @@ describe('GET /api/topics', () => {
                 expect(response.body.msg).toBe(`Username ${userData}, not found!`)
             });
         })
-    })
\ No newline at end of file
+    })
+    describe('PATCH /api/articles/:article_id', () => {
+        test('PATCH /api/articles/:article_id, should update article by id, with provided positive integer', () => {
+            const newVote = 10
+            const update = { inc_votes: newVote };
+
+            return request(app)
+            .patch("/api/articles/1")
+            .expect(200)
+            .send(update)
+            .then((response) => {
+                const updatedArticle = response.body.article
+                expect(updatedArticle.votes).toBe(110);
+            })
+        });
+        test('PATCH /api/articles/:article_id, should update article by id, with provided negative integer', () => {
+            const newVote2 = -100;
+            const update2 = { inc_votes: newVote2 };
+
+            return request(app)
+            .patch("/api/articles/1")
+            .expect(200)
+            .send(update2)
+            .then((response) => {
+                const updatedArticle2 = response.body.article
+                expect(updatedArticle2.votes).toBe(0);
+            })
+        });
```

Comment:
As we ask for the response to be the whole article it would be good for the test to check the shape of all the whole returned article not sure just votes 
But great that you've got two happy path tests for positive and negative increments 🙌